CHANGELOG.md

```diff
@@ -1,21 +1,26 @@
-# 1.1.0 (2020-09-04)
```

Comment:
This file normally gets generated by `oh-release`, so you wouldn't need to / want to edit it manually.

Can you revert?

Reply:
There we go! Let me know if that looks better! I rebased that faux pas :smile: